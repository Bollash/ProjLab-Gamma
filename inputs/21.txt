Create Space
Add Asteroid Uran
Mod Layer 0 0
Add Ufo 0
Countdown 0
Countdown 0
Countdown 0
Countdown 0
Countdown 0
Countdown 0
Countdown 0
Countdown 0
Countdown 0
Status All