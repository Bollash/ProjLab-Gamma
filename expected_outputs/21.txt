Space létrehozva
A nyersanyaggal teli aszteroida hozzá lett adva a space-hez.
Sikeres volt a ufóhozzáadás
Az adott aszteroida kérge a kapott értékre lett állítva.
A napközeli számláló eggyel csökkent.
A napközeli számláló eggyel csökkent.
Napközelbe került az aszteroida.
Az urán számlálója eggyel nőtt.
A napközeli számláló eggyel csökkent.
A napközeli számláló eggyel csökkent.
Napközelbe került az aszteroida.
Az urán számlálója eggyel nőtt.
A napközeli számláló eggyel csökkent.
A napközeli számláló eggyel csökkent.
Napközelbe került az aszteroida.
Az aszteroida felrobbant.
Space
0
10
10
False
0
0